clients/drcachesim/drpt2trace/drpt2trace.cpp

```diff
@@ -377,21 +449,113 @@ main(int argc, const char *argv[])
         config.pt_config.cpu.family != 0 ? CPU_VENDOR_INTEL : CPU_VENDOR_UNKNOWN;
 
     /* Convert the PT raw trace to DR IR. */
+    std::unique_ptr<pt_iscache_autoclean_t> shared_iscache(
+        new pt_iscache_autoclean_t(pt_iscache_alloc(nullptr)));
     std::unique_ptr<pt2ir_t> ptconverter(new pt2ir_t());
-    if (!ptconverter->init(config)) {
-        std::cerr << CLIENT_NAME << ": failed to initialize pt2ir_t." << std::endl;
+    std::unique_ptr<drir_t> drir(new drir_t(GLOBAL_DCONTEXT));
+
+    std::unique_ptr<uint8_t[]> buffer = std::unique_ptr<uint8_t[]>(nullptr);
+    size_t buffer_size = 0;
```

Comment:
So we're passing a buffer_size of zero to load_pt_raw_file? Add comment or fix as needed.

Reply:
Utilize a vector to store raw PT data readed from the PT file.